### PR TITLE
Add the options to disable different filters in robot detection

### PR DESCRIPTION
**Pull request by JonasHolz**
## Introduced Changes

Adds options to disable filtering in robot detection
Disables use_robot_detection_measurements 

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

Look at robot detection and compare the results with enabled and disabled filters

### CODE REVIEW

File: etc/configuration/default.json [hunk 1]
```diff
@@ -260,7 +260,7 @@
     "initial_covariance": [0.25, 0.25],
     "measurement_count_threshold": 10,
     "use_feet_detection_measurements": true,
-    "use_robot_detection_measurements": true,
+    "use_robot_detection_measurements": false,
```
Comment (julianschuler) [inline etc/configuration/default.json:263]: Please also add the new parameters in the `default.json` file.

**Review by julianschuler — CHANGES_REQUESTED** [review]
One thing is missing, the rest LGTM!

File: etc/configuration/default.json (hunk 1 above)
Comment (JonasHolz) [reply etc/configuration/default.json]: Done!

File: crates/vision/src/robot_detection.rs [hunk 2]
```diff
@@ -94,15 +98,23 @@ impl RobotDetection {
...
-            context.allowed_projected_robot_height,
-        );
+        if *context.enable_filter_by_pixel_position {
+            filtered_detections = filter_by_pixel_position(
+                filtered_detections,
+                *context.lowest_bottom_pixel_position,
+            );
+        }
+
+        if *context.enable_filtered_detections {
```
Comment (schmidma) [inline crates/vision/src/robot_detection.rs:110]: should be called `enable_filter_by_size`

File: etc/configuration/default.json [hunk 3]
```diff
@@ -153,6 +153,8 @@
   "robot_detection": {
     "vision_top": {
       "enable": true,
+      "enable_filtered_detections": true,
+      "enable_filter_by_pixel_position": true,
```
Comment (schmidma) [inline etc/configuration/default.json:157]: Disable for now by default

**Review by schmidma — CHANGES_REQUESTED** [review]

File: etc/configuration/default.json (hunk 1 above)
Comment (JonasHolz) [reply etc/configuration/default.json]: Done!

File: crates/vision/src/robot_detection.rs (hunk 2 above)
Comment (JonasHolz) [reply crates/vision/src/robot_detection.rs]: Done!

File: etc/configuration/default.json (hunk 3 above)
Comment (JonasHolz) [reply etc/configuration/default.json]: Done!

**Review by schmidma — APPROVED** [review]

**Review by julianschuler — APPROVED** [review]

**Pull request merged**